src/Microsoft.Restier.EntityFramework/DbDomain.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.Restier.EntityFramework
     /// <summary>
     /// Represents a domain over a DbContext.
     /// </summary>
+    [CLSCompliant(false)]
```

Comment:
EF7 DbContext type is not CLS compliant, and that makes a type references it also not compliant. This has been modified to conditionally with EF7.